libs/deprecations.liq

```diff
@@ -35,4 +36,11 @@ end
 def quote(s) =
   deprecated("quote", "string.quote")
   string.quote(s)
-end
\ No newline at end of file
+end
+
+# Deprecated: use mksafe and playlist instead.
+# @flag hidden
+def playlist.safe(~id, ~mime_type, ~mode, ~on_track, ~prefix, ~reload, ~reload_mode, uri)
+  deprecated("playlist.safe", "")
```

Comment:
The deprecation message is misleading here. *de facto* `playlist.safe` has already been removed..

Reply:
I don't understand this either: _deprecated_ means that the function should not be used anymore, but we are still defining it (precisely with this definition).

Reply:
Usually, deprecation messages means that the function is still here but may be removed. But in this case, it's here but not doing what the old function was doing so this will be confusing and scripts using it will break or behave differently.

Reply:
Well it's more or less an equivalent implementation...